### ISSUE TITLE
Learning rate not updating as it should - efficientnets

### ISSUE THREAD

**Issue by GPU-Shuttle** [DeepLabCut/DeepLabCut#1191]
Hi,
So, we managed to run the efficientnet-b6 model in ticket #1144 on an NVIDIA A100 GPU (40GB mem), instead of a P100. A resnet_152 model was previously trained with the same training parameters as below, properly. However, switching to the efficientnet-b6, it seems that the default learning rate of 0.0005 is not being updated as it should, and hence the loss seems to be on an extremely slow converging path (see output below)!

Any suggestion on what parameter to amend, please?

Thanks :-) 

--------------------------------------------------------------------------------------------
Training parameters
--------------------------------------------------------------------------------------------
'alpha_r': 0.02,
 'batch_size': 1,
 'crop_pad': 0,
 'cropratio': 0.4,
 'dataset_type': 'imgaug',
 'decay_steps': 30000,
 'deterministic': False,
 'display_iters': 1000,
 'fg_fraction': 0.25,
 'global_scale': 0.8,
 'intermediate_supervision': False,
 'intermediate_supervision_layer': 12,
 'location_refinement': True,
 'locref_huber_loss': True,
 'locref_loss_weight': 0.05,
 'locref_stdev': 7.2801,
 'log_dir': 'log',
 'lr_init': 0.0005,
 'max_input_size': 1500,
 'mean_pixel': [123.68, 116.779, 103.939],
 'min_input_size': 64,
 'mirror': False,
 'multi_step': [[0.005, 10000],
                [0.02, 430000],
                [0.002, 730000],
                [0.001, 1030000]],
 'net_type': 'efficientnet-b6',
 'num_joints': 12,
 'optimizer': 'sgd',
 'pairwise_huber_loss': False,
 'pairwise_predict': False,
 'partaffinityfield_predict': False,
 'pos_dist_thresh': 17,
 'regularize': False,
 'rotation': 25,
 'rotratio': 0.4,
 'save_iters': 50000,
 'scale_jitter_lo': 0.5,
 'scale_jitter_up': 1.25,
 'scoremap_dir': 'test',
 'shuffle': True,
 'stride': 8.0,
 'weigh_negatives': False,
 'weigh_only_present_joints': False,
 'weigh_part_predictions': False,
 'weight_decay': 0.0001}

--------------------------------------------------------------------------------------------
output for efficientnet-b6
--------------------------------------------------------------------------------------------
2021-04-10 02:53:38 iteration: 1000 loss: 0.0988 lr: 0.0004986578715033829
2021-04-10 03:10:35 iteration: 2000 loss: 0.0249 lr: 0.0004946461995132267
…
…
…
2021-04-10 03:58:33 iteration: 9000 loss: 0.0183 lr: 0.00039900740375742316
2021-04-10 04:03:32 iteration: 10000 loss: 0.0179 lr: 0.00037750002229586244
2021-04-10 04:08:48 iteration: 11000 loss: 0.0182 lr: 0.000354650488588959
…
…
…
2021-04-11 12:21:47 iteration: 429000 loss: 0.0163 lr: 1.0000000656873453e-05
2021-04-11 12:26:16 iteration: 430000 loss: 0.0160 lr: 1.0000000656873453e-05
2021-04-11 12:30:51 iteration: 431000 loss: 0.0164 lr: 1.0000000656873453e-05
…
…
…
2021-04-12 11:28:56 iteration: 729000 loss: 0.0161 lr: 1.0000000656873453e-05
2021-04-12 11:33:30 iteration: 730000 loss: 0.0159 lr: 1.0000000656873453e-05
2021-04-12 11:38:06 iteration: 731000 loss: 0.0160 lr: 1.0000000656873453e-05

**Comment by AlexEMG** [comment]
That looks correct -- i.e. 

```
'multi_step': [[0.005, 10000],
[0.02, 430000],
[0.002, 730000],
[0.001, 1030000]],
```
is not used for EfficientNets -- we found that cosine decay performed really well. See supplementary material in the [WACV paper](https://openaccess.thecvf.com/content/WACV2021/html/Mathis_Pretraining_Boosts_Out-of-Domain_Robustness_for_Pose_Estimation_WACV_2021_paper.html) for more details on how hese numbers impacted our results! 

The relevant parameters are outlined in the default pose_config file:
https://github.com/DeepLabCut/DeepLabCut/blob/master/deeplabcut/pose_cfg.yaml#L117

They are:
**lr_init: 0.0005
decay_steps: 30000
alpha_r: 0.02**
